frontend/src/metabase/palette/components/PaletteResultsList.unit.spec.tsx

```diff
@@ -0,0 +1,132 @@
+import { Route } from "react-router";
```

Comment:
note to self: probably should move this into PaletteResultItem.unit.spec.tsx, and the setup function here can become setupList there